### PR TITLE
changed instructions for how to build

### PR DESCRIPTION
**Pull request by heckj**
mostly line-end wrapping changes, but made the content around building the default images more explicit.

### CODE REVIEW

**Comment by heckj** [comment]
@benbp what's your thought on how to run these as a part of an overall setup, and then make them available for the instructions at https://github.com/rackhd/rackhd ? Extend the vagrant instance to build them locally in that instance? travis builds in on-imagebuilder that stuffs at least default results into binary and use the files from there?

File: README.md [hunk 1]
```diff
@@ -65,29 +68,31 @@ Getting started
 ### Building images
 
 To build images, define an imagebuilding playbook (see example.yml for an example) or
```
Comment (heckj) [inline README.md:70]: @benbp this is the meat of this change

File: README.md [hunk 2]
```diff
@@ -41,7 +41,10 @@ and/or shell scripts, and is often under 10mb in size and takes under a minute t
 **provisioner**
 
 An ansible role used to specify changes that should be made to the filesystem
-of an initrd, base image or overlay (e.g. extra packages, scripts, files).
+of an initrd, base image or overlay (e.g. extra packages, scripts, files). An
```
Comment (heckj) [inline README.md:44]: @benbp this is the meat of this change

**Comment by benbp** [comment]
@heckj These changes should be made in https://github.com/RackHD/docs/blob/master/docs/monorail/microkernel.rst too. I think the plan is to deprecate this README and just link to that.

**Comment by benbp** [comment]
Re your question: yeah I think if we can get around the legal issues of distribution, it'd be great to just download the latest builds, otherwise, yeah we'll want to set it up as part of the ansible provisioning step for the vagrant setup.

**Comment by benbp** [comment]
:+1:

**Comment by heckj** [comment]
Ah - very cool, I'll go read through what's in the RTD site and update this PR to be a change to point to that information to locate the docs in a single place. Honestly won't be able to tackle until tomorrow earliest though

**Pull request merged**